examples/Makefile

```diff
@@ -60,14 +60,14 @@ EXAMPLES = \
     strings \
     subroutine_args \
     subroutine_contains_issue101 \
+    f2py_string_input \
     issue299_directc_nested_functions \
     recursive_type \
     type_bn \
     type_check
 
 # TODO: Fix these failing examples
```

Comment:
The underlying issue is still present. I think that keeping the TODO as is makes it more clear that there is a pending issue.